Packs/SilentPush/Integrations/SilentPush/SilentPush.py

```diff
@@ -3912,6 +4136,256 @@ def screenshot_url_command(client: Client, args: dict[str, Any]) -> CommandResul
     )
 
 
+@metadata_collector.command(
+    command_name="silentpush-add-feed",
+    inputs_list=ADD_FEED_INPUTS,
+    outputs_prefix="SilentPush.AddFeed",
+    outputs_list=ADD_FEED_OUTPUTS,
+    description="This command add the new feed",
+)
+def add_feed_command(client: Client, args: dict[str, Any]) -> CommandResults | dict:
+    """
+    Command handler for adding new feeds.
+
+    Args:
+        client (Client): SilentPush API client instance.
+        args (Dict[str, Any]): Command arguments, must include 'name' and 'type' key.
+
+    Returns:
+        CommandResults: JSON response of added feed.
+    """
+
+    name = args.get("name")
+    if not name:
+        raise ValueError("Name is required")
+
+    feed_type = args.get("type")
+    if not feed_type:
+        raise ValueError("Type is required")
+
+    result = client.add_feed(args)
+
+    return CommandResults(
+        outputs_prefix="SilentPush.AddFeed",
```

Comment:
not sure AdFeed is a good context key path, if you plan to return data of the added feed, then Feed should be good